esphome/nspanel_esphome_core.yaml

```diff
@@ -854,7 +854,7 @@ display:
             case 22:  // fan
               switch (component_id) {
                 case 17:  // bt_oscillate
-                  ha_call_service->execute("fan.oscillate", "", "toggle", detailed_entity->state.c_str());
```

Comment:
I left this "toggle" option intentionally, so we can keep space for true/false in the future, but I'm fine about removing it for now and we implement it again if needed in the future.